backend/src/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+export enum GameState {
```

Comment:
now saying we should necessarily, but maybe we could just use 0 and 1 to represent the different game states rather than strings?